javascript/react/src/components/ArqueroDetailsList/renderers/DefaultColumnHeader.tsx

```diff
@@ -30,6 +41,12 @@ export const DefaultColumnHeader: React.FC<DefaultColumnHeaderProps> = memo(
 			isIconOnly,
 		} = column
 
+		const hostStyles: Partial<ITooltipHostStyles> = {
+			root: { display: 'inline-block' },
+		}
+		const tooltipId = useId('tooltip')
+		const calloutProps = { gapSpace: 0 }
```

Comment:
This const can be outside of the function since it does not have any dependencies